### ISSUE TITLE
Handle existing SDC creator statements that just have the username in the string

### ISSUE THREAD

**Issue by alexwlchan** [Flickr-Foundation/flickypedia#430]
e.g. https://commons.wikimedia.org/wiki/File:Apple_orchard_in_Tasmania.jpg

**Comment by alexwlchan** [comment]
It turns out that was a bad example, because the names were slightly different – "Apple and Pears" vs "Apple and Pear". I fixed it by hand, and I'm keeping an eye out for other examples.

**Comment by alexwlchan** [comment]
I'm going to close this for now, because I'm less sure this is actually a widespread issue.

**Comment by alexwlchan** [comment]
Here's an example which looks more promising (selected at random): 
![](https://commons.wikimedia.org/wiki/File:John_Altman_7.jpg)

**Comment by alexwlchan** [comment]
Except that file is a fun example of where the `realname` field on Flickr is displayed, but not populated!

```console
$ fluser "https://www.flickr.com/photos/mangakamaidenphotography/"
NSID:     24960504@N06
username: MangakaMaiden Photography
realname:
URL:      https://www.flickr.com/photos/mangakamaidenphotography/
```

but the photo page shows "[Michelle Claire Woolnough](https://www.flickr.com/photos/mangakamaidenphotography/)".

**Comment by alexwlchan** [comment]
Fixed in #446.
